### PR TITLE
OVH to OVHcloud

### PR DESCRIPTION
**Pull request by mhurtrel**
Company was rebranded in 2019

### CODE REVIEW

**Comment by yanndegat** [comment]
hi @mhurtrel 

maybe we can change only references to the company, but not the name of the provider 

otherwise, we would have to rename a lot a stuff, including this repo, and its hosting organization

what do you think ?

**Comment by mhurtrel** [comment]
@yanndegat yup fully aligned, just changed the text in the documentation comments but I agree with should keep the ovh/ovh resources naming, URLs to ensure keeping links and do not break anything

File: website/docs/index.html.markdown [hunk 1]
```diff
@@ -3,19 +3,19 @@ layout: "ovh"
 page_title: "Provider: OVH"
 sidebar_current: "docs-ovh-index"
 description: |-
-  The OVH provider is used to interact with the many resources supported by OVH. The provider needs to be configured with the proper credentials before it can be used.
+  The OVH provider is used to interact with the many resources supported by OVHcloud. The provider needs to be configured with the proper credentials before it can be used.
 ---
 
-# OVH Provider
+# OVHcloud Provider
```
Comment (yanndegat) [inline website/docs/index.html.markdown:9]: should be kept as OVH

File: website/docs/index.html.markdown [hunk 2]
```diff
@@ -3,19 +3,19 @@ layout: "ovh"
...
 
-The OVH provider is used to interact with the many resources supported by OVH. 
+The OVHcloud provider is used to interact with the many resources supported by OVHcloud. 
 The provider needs to be configured with the proper credentials before it can be used.
```
Comment (yanndegat) [inline website/docs/index.html.markdown:12]: the OVH provider is used to interact with the many resources supported by OVHcloud.

File: website/docs/index.html.markdown [hunk 3]
```diff
@@ -25,7 +25,7 @@ These parameters can be configured directly in the provider block as shown herea
 
 
 ```hcl
-# Configure the OVH Provider
+# Configure the OVHcloud Provider
 provider "ovh" {
```
Comment (yanndegat) [inline website/docs/index.html.markdown:29]: should be kept as OVH

**Pull request merged**